collect_app/src/main/java/org/odk/collect/android/views/MediaLayout.java

```diff
@@ -142,9 +142,9 @@ public void playVideo() {
 
             Intent i = new Intent("android.intent.action.VIEW");
             i.setDataAndType(Uri.fromFile(videoFile), "video/*");
-            try {
+            if (i.resolveActivity(getContext().getPackageManager()) != null) {
```

Comment:
That is really strange. It is indeed [here](https://github.com/opendatakit/collect/blob/master/collect_app/src/main/java/org/odk/collect/android/views/MediaLayout.java#L145). And the target branch is set to opendatakit/master. That is concerning. I guess we'll just have to hope the merge works!